### PR TITLE
chore: new project abr-testing

### PR DESCRIPTION
**Pull request by sfoster1**
Add a new project for ABR test files so they do not have to adhere to the requirements that hardware-testing does, and vice versa.

### CODE REVIEW

File: .github/workflows/abr-testing-lint-test.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+name: 'abr-testing lint/test'
+on:
+  push:
+    paths:
+      - 'Makefile'
+      - 'abr-testing/**'
+      - 'scripts/**/*.mk'
+      - 'scripts/**/*.py'
+      - '.github/workflows/abr-testing-lint-test.yaml'
+      - '.github/actions/ython/**'
```
Comment (SyntaxColoring) [inline .github/workflows/abr-testing-lint-test.yaml]:
```suggestion
      - '.github/actions/python/**'
```

File: abr-testing/setup.py
Comment (SyntaxColoring) [inline abr-testing/setup.py:1]: Should this have a setup.py if we don't intend on packaging and publishing it anywhere?

**Review by SyntaxColoring — APPROVED** [review]
LGTM when CI passes.

File: abr-testing/setup.py
Comment (sfoster1) [reply abr-testing/setup.py:1]: yes because maybe we will in the future and it's easier. I want the uniformity

**Pull request merged**